### DISCUSSION TITLE
How can I set additional label for broker pods?

### DISCUSSION THREAD

**Discussion by lnthai2002** [arkmq-org/arkmq-org-broker-operator#902] [Q&A] [answered]
Hello, I need to add some special labels to the broker pod. Is it possible?

**Comment by brusdev** [comment, marked as answer]
@lnthai2002 you can add labels to the broker pod by using the field spec.deploymentPlan.labels, for further details see the documentation at https://github.com/artemiscloud/activemq-artemis-operator/blob/main/docs/help/operator.md#configuring-labels-and-annotations

**Comment by lnthai2002** [comment]
Thank you
